python/ray/tune/search/optuna/optuna_search.py

```diff
@@ -305,7 +309,7 @@ def define_search_space(trial: optuna.Trial):
         )
         tuner.fit()
 
-    .. versionadded:: 0.8.8
+    .. versionadded:: 0.8.9
```

Comment:
cc @justinvyu, @matthewdeng, Could you remind me if this version number should be modified like this?

Reply:
We should leave it at `0.8.8`, this is the original version of Ray that this class was added.